living_documentation_regime/README.md

```diff
@@ -0,0 +1,358 @@
+    - Examples: `ReportingTopic`, `UserManagementTopic`, `SecurityTopic`.
+  - **NoTopic:** Indicates that the ticket does not align with a specific topic, based on the editor's discretion.
+- **Type**:
+  - **DocumentedUserStory:** Describes a user-centric functionality or process, highlighting its purpose and value.
+    - Encompasses multiple features, capturing the broader goal from a user perspective.
+  - **DocumentedFeature:** Details a specific feature, providing a breakdown of its components and intended outcomes.
+    - Built from various requirements and can relate to multiple User Stories, offering an in-depth look at functionality.
+  - **DocumentedRequirement:** Outlines individual requirements or enhancements tied to the feature or user story.
+- **Issue States**:
+  - **Upcoming:** The feature, story, or requirement is planned but not yet implemented.
+  - **Implemented:** The feature or requirement has been completed and is in active use.
+  - **Deprecated:** The feature or requirement has been phased out or replaced and is no longer supported.
+
+**DocumentedUserStory** and **DocumentedFeature** serve as **Epics**, whereas **DocumentedRequirement** represents specific items similar to feature enhancements or individual requirements.
+
+### Hosting Documentation Tickets in a Solo Repository
+
+Using a dedicated repository solely for documentation tickets provides multiple advantages:
+- **Streamlined Management:** This avoids cross-project conflicts and board exclusions and enables specialized templates solely for documentation purposes.
+- **Focused Access Control:** This allows a small team to manage and edit documentation without interference, maintaining high-quality content.
+- **Optimized Data Mining:** Supports easier and more efficient data extraction for feedback and review cycles through Release Notes.
+- **Implementation Reflection:** Mirrors elements from the implementation repositories, providing a high-level knowledge source that is valuable for both business and technical teams.
+- **Release Notes Integration:** Documentation can evolve based on insights from release notes, serving as a dynamic feedback loop back to the documentation repository.
+
+---
+## Living Documentation Regime Features
+
+### Data Mining from GitHub Repositories
+
+This feature allows you to define which repositories should be included in the living documentation process. By specifying repositories, you can focus on the most relevant projects for your documentation needs.
+
+- **Default Behavior**: By default, the action will include all repositories defined in the repositories input parameter. Each repository is defined with its organization name, repository name, and query labels.
+
+### Data Mining from GitHub Projects
+
+This feature allows you to define which projects should be included in the living documentation process. By specifying projects, you can focus on the most relevant projects for your documentation needs.
```

Comment:
Here, I am missing information on what action has to be activated.
After activation, we can talk about default and non-default behavior.

Reply:
Hopefully added at: